docs/getting-started/getting-started.md

```diff
@@ -10,5 +10,6 @@ Install Dragonfly
 
 - [Install with Docker](./docker.md)
 - [Install with Docker Compose](./docker-compose.md)
+- [Install Dragonfly Kubernetes Operator](./kubernetes-operator.md)
```

Comment:
How about modifying "Install with Kubernetes" below to "Install on Kubernetes with Helm Charts" or "Install with Kubernetes Helm Charts" ?